Makefile

```diff
@@ -118,7 +118,7 @@ integration-test:
 
 .PHONY: ci-integration-test
 ci-integration-test: $(GO_JUNIT_REPORT)
-	GO_TEST_FLAGS="-coverprofile=coverage.txt -v" make -s integration-test | go-junit-report;
+	GO_TEST_FLAGS="-p 256 -coverprofile=coverage.txt -v" make -s integration-test | go-junit-report;
```

Comment:
Why 256? [More is not always better.](https://en.wikipedia.org/wiki/Amdahl%27s_law)

My intuition says that ideally the `-p` is set to the number of logical cores, which is the default.

If anything, it sounds like we should be setting `GOMAXPROCS` or `-parallel`, but the defaults for those values (also the number of logical cores) also sound sane to me.

I would not adjust these knobs without benchmarks and good evidence that this is better.

See also https://github.com/golang/go/issues/11521 and `go help testflag`.